Packs/Smokescreen_IllusionBLACK/IncidentTypes/incidenttype-IllusionBLACK.json

```diff
@@ -0,0 +1,23 @@
+{
+	"autorun": true,
+	"closureScript": "",
+	"color": "#9eabcd",
+	"days": 0,
+	"daysR": 0,
+	"default": false,
+	"disabled": false,
+	"hours": 0,
+	"hoursR": 0,
+	"id": "IllusionBLACK",
+	"locked": false,
+	"name": "IllusionBLACK",
+	"playbookId": "2e000099-7f01-4a14-88a0-062f633ee89f",
```

Comment:
Please update this to use your playbook ID

Reply:
updated playbook id